liftwing/models/revscoring.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, Dict
+
+import requests
+from pydantic import BaseModel, Field
+
+from .liftwing_model import LiftwingModel, ModelMetadata
+
+
+class RevscoringPayload(BaseModel):
+    rev_id: int = Field(...,
+                        title="Revision ID of the edit",
+                        description="The revision id for the wiki",
+                        gt=0)
```

Comment:
The revscoring models also support an optional `extended_output` boolean parameter:
https://api.wikimedia.org/wiki/Lift_Wing_API/Reference/Get_revscoring_damaging_prediction#POST_Parameters

Reply:
Added an optional field so that we can also use `extended_output`. A pydantic Field is made optional by not ommitting the three dots in the Field constructor (`...`)